nodeup/pkg/model/tests/golden/minimal/cluster.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       backupStore: memfs://clusters.example.com/minimal.example.com/backups/etcd-events
   kubelet:
     anonymousAuth: false
-  kubernetesVersion: v1.18.0
+  kubernetesVersion: https://storage.googleapis.com/kubernetes-release/release/v1.18.0
```

Comment:
Why the full URL?

Reply:
This activates side-loading and the `-amd64`/`-arm64` suffix for the images.

Reply:
I'm concerned about this invalidating a lot of the golden tests, as they would no longer cover the common case of no sideloading. How would our tests detect a bug that is masked by sideloading? Our e2e tests certainly wouldn't find it; the only choices would be unit, integration, or beta.

Reply:
It's just an image name, nothing special. I could create a new copy of the tests for this if you feel so strongly, or just drop this commit.

Reply:
If a new copy of the tests for this would give you adequate coverage then I'd prefer that approach. Or would you need to duplicate the whole row of golden tests to make sure each builder handles sideloading correctly?

Reply:
We pretty much need to duplicate the `minimal` dir to a new one called `side-loading`.
Just pushed this as a change.

Reply:
Hrm, this is what I meant by "need to duplicate the whole row of golden tests". I concede my concern about not covering the no-sideloading case is probably not worth the resulting cost. You can rip that out if you like. Perhaps a future PR could make the AMD/ARM/no-sideload matrix be table-driven, with text repacement to the golden file in order to form the expected output.

Reply:
Would it be ok to keep it as is (duplicated) for now and will try to de-duplicate it in a future PR?
Except for the fact that we get to have duplicated `cluster.yaml` file things don't change so much.